RocketChat/Rocket.Chat
# fix: Show badge for threads direct mentions

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce


- Create a room
- Go to notification preferences in that room
- Disable "Mark as Unread"
- Enable "Show badge for mentions"
- From 2nd user, mention 1st user in a thread

Badge on sidebar doesn't appear for direct mention for threads.

## Further comments

TC-765
